client/app/pages/queries-list/QueriesListEmptyState.jsx

```diff
@@ -19,15 +21,19 @@ export default function QueriesListEmptyState({ page, searchTerm, selectedTags }
     case "archive":
       return <BigMessage message="Archived queries will be listed here." icon="fa-archive" />;
     case "my":
-      return (
-        <div className="tiled bg-white p-15">
+      const my_msg = currentUser.hasPermission("create_query") ? (
```

Comment:
Maybe we can extract this to a component? Feels like there's some duplication. Not sure if there will be enough re-use to justify it, but I wanted to report my first impression.

Reply:
That's definitely a good idea, but IMHO it shouldn't block this PR - we can do this refactoring separately